--- vicious-1.0.26-orig/init.lua	2010-03-15 13:12:21.597605325 +0100
+++ vicious-1.0.26/init.lua	2010-03-15 13:12:31.220937702 +0100
@@ -115,7 +115,7 @@
     end
 
     if widget.add_value ~= nil then
-        widget:add_value(data and tonumber(data) / 100)
+        widget:add_value(data and tonumber(data))
     elseif widget.set_value ~= nil then
         widget:set_value(data and tonumber(data) / 100)
     else
